File: SCADA/SCADA/ClientApp/src/app/model/TagDTO.ts
export interface TagDTO {
    type: string,
    id: number | null,
    name: string,
    ioAddress: string,
    description: string,
    value: number,
    driver: string | null,
    scanTime: number | null,
    isScanOn: boolean | null,
    lowLimit: number | null,
    highLimit: number | null,
    units: string | null,
    initialValue: number | null
}

File: SCADA/SCADA/DTOS/TagDTO.cs
﻿namespace SCADA.DTOS
{
    public class TagDTO
    {
        public string Type { get; set; }
        public string Name { get; set; }
        public string Description { get; set; } 
        public string Driver { get; set; }
        public string IOAddress { get; set; }
        public double? InitialValue { get; set; }
        public int? ScanTime { get; set; }
        public bool? IsScanOn { get; set; }
        public double? LowLimit { get; set; }
        public double? HighLimit { get; set; }
        public string? Units { get; set; }
    }
}
